src/styles/global/_custom-bootstrap.scss

```diff
@@ -8,47 +8,13 @@ $light: $white;
 //font-size overrides
 $small-font-size: 65%;
 
-// button overrides
-$btn-border-radius: 0;
-$btn-padding-y: 0.267rem;
-$btn-padding-x: 0.8rem;
-
 // modal overrides
 $modal-content-border-radius: 0;
 
-// pagination overrides
-$pagination-color: #222;
-$pagination-active-bg: $selected;
-$pagination-active-color: $white;
-$pagination-hover-bg: $selected;
-$pagination-hover-color: $white;
-$pagination-border-color: $udb-grey;
-$pagination-focus-box-shadow: none;
-$pagination-padding-y: 0.44rem;
-$pagination-padding-x: 0.84rem;
-
 // form-input overrides
 $input-border-radius: 0;
 $input-border-radius-lg: 0;
 $input-border-radius-sm: 0;
 
 @import '../../../node_modules/bootstrap/scss/bootstrap.scss';
 @import '../../../node_modules/bootstrap/scss/variables.scss';
-
-// button overrides
-.btn-outline-secondary {
-  color: #333;
-  background-color: $white;
-
-  &:hover {
-    background-color: #e6e6e6;
-    border-color: #adadad;
-  }
-}
-
-//alert overrides
-.alert-info {
-  background-color: #d9edf6;
-  border-color: #bce8ef;
-  color: #3e88ab;
```

Comment:
Maybe we could also store these colors in the theme as we're doing with btn-outline-secondary